packages/suite-desktop/src/modules/coinjoin.ts

```diff
@@ -85,6 +94,8 @@ export const init: Module = ({ mainWindow }) => {
         const dispose = () => {
             unregisterBackendProxy();
             unregisterClientProxy();
+            logger.info('coinjoin', 'Stopping (app quit)');
+            coinjoin.stop();
         };
```

Comment:
Seams like good idea to stop it when app refreshes and quit. Since coinjoin account also disappears when app refreshes. 
With bridge we do different. We keep it running if app refreshes because it is for sure going to be needed. But it is not the case of coinjoin middleware.

Reply:
The fact the CJ stays when a wallet is remembered is a feature or a bug? The process should be stopped regardless, I just wanted to clarify.

Reply:
So when wallet is remembered the CJ account stays? I did not know that, if that is the case I think the process should not be stopped when refreshing, otherwise complicates the thing too much.
@szymonlesisz Could you clarify?

Reply:
it should be:
- disabled by suite (by calling "disable" method) while forgetting the accounts
- disabled on refresh, it will be enabled again when needed (by calling "createInstance")